deploy-agent/tox.ini

```diff
@@ -1,10 +1,10 @@
 [tox]
-envlist=py27,py3
+envlist=py37
```

Comment:
i think we still need to support py2 for a while. so it is better to keep both py2 and py3